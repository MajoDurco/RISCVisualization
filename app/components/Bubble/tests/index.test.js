import React from 'react';
import renderer from 'react-test-renderer'

import Buble from '../index';

describe('<Buble />', () => {
  it('Snapshots bubble component', () => {
    const buble = renderer.create(
      <Buble height={'100px'} width={'100px'} />
    ).toJSON()
    expect(buble).toMatchSnapshot()
  })
})
