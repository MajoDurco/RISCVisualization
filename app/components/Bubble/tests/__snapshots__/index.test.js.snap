exports[`<Buble /> Snapshots bubble component 1`] = `
<div
  className="kJXBvM"
  height="100px"
  width="100px" />
`;
